cypress/integration/dashboard_spec.ts

```diff
@@ -42,11 +43,23 @@ describe('The Dashboard', () => {
     cy.getBySel('swap-assets-button').click()
     // cy.getBySel('trade-rate-quote').should('have.text', '1 FOX = 0.00011 ETH') // TODO - mock response and test here
     cy.getBySel('trade-preview-button').should('be.disabled')
+
+    // Clicking 'swap-assets-button' will cause XHR requests, lets wait for
+    // those requests to complete since the 'trade-preview-button' will have
+    // the text "Loading..." (displays a spinner)
+    // cy.intercept({ method: 'GET', url: '/swap/**'}).as('getSwap')
+    // cy.intercept({ method: 'GET', url: '**/account/**'}).as('getAccount')
+    // cy.wait(['@getSwap', '@getAccount'], { timeout: 60000 })
+    cy.intercept({ method: 'GET' }).as('getReqs')
+    cy.wait(['@getReqs'])
+
     cy.getBySel('token-row-sell-max-button').click()
+    cy.waitForAllGetReqs()
     // TODO@0xApotheosis - this timeout won't be necessary once external request bounty complete
-    cy.getBySel('trade-preview-button').should('have.text', 'Insufficient Funds', {
-      timeout: 30000,
-    })
+    // cy.getBySel('trade-preview-button', {
+    //   timeout: 30000,
+    // }).should('have.text', 'Insufficient Funds')
```

Comment:
We can remove this, yeah?

Reply:
Yes we can. While I am here, I will say that the Cypress docs recommend using `contains(...)` instead of `.should('have.text', ...)`

Reply:
I'm on board with swapping those instances out 🚀

Reply:
Will go ahead and take care of these now. :)